anyblok/declarations.py

```diff
@@ -151,38 +151,50 @@ def wrapper(self):
             return wrapper
 
 
-def cache(size=128):
-    autodoc = """
-    **Cached method** with size=%(size)s
-    """ % dict(
-        size=size
-    )
+class cache:
```

Comment:
because it became a class I would start with uppercase:
```suggestion
class Cache:
```

Reply:
I want to keep the man behaviour

Reply:
I change the name and add a wrapper with old names and deprecated warning